### DISCUSSION TITLE
Buttons for replying, editing message, when you hover a message

### DISCUSSION THREAD

**Discussion by ghost** [stoatchat/discussions#156] [Feature Suggestions]
Basically I think it would be a nice feature to have these buttons like in Discord.

![image](https://user-images.githubusercontent.com/68067125/134640880-e57cf266-c018-4886-a94c-b2a4b47d04dc.png)

Like this.

Once reactions are implemented, they could also be placed there.

**Comment by oomfiee** [comment]
and with syntax/Markdown Text 
![44444444444444444444f](https://user-images.githubusercontent.com/80048740/134641056-9e1c48df-4d7e-4ba2-8fdf-4f3c3b6eb9a4.PNG)
